darwin/importer/formats/nifti.py

```diff
@@ -16,8 +16,8 @@
     from scipy.ndimage import zoom
 except ImportError:
     import_fail_string = """
-    You must install darwin-py with pip install nibabel connected-components-3d
-    in order to import with using nifti format
+    You must install darwin-py with pip install darwin-py\[medical]
+    in order to export using nifti format
```

Comment:
This came up recently from a user - It's better to let the optional `[medical]` do the work instead